web/server/README.md

```diff
@@ -98,21 +98,21 @@ openssl speed rsa2048 rsa4096
 
 ### Select TLS version
 
-Since version 1.21 Netdata accepts `tls key` and `tls certificate` as replacement of the options `ssl key` and `ssl certificate`, 
- but you do not need to change your `netdata.conf` case you are using old `ssl` variables, because old options are kept due compatibility.
+Beginning with version 1.21, Netdata accepts `tls key` and `tls certificate` as replacement of the settings `ssl key` and `ssl certificate`. 
 
-Netdata also brings two new options related to TLS, now you can specify the TLS version and the ciphers that you want to use:
+You do not need to change your `netdata.conf` if you are using the previous `ssl` settings, because these are kept for backward compatibility.
+
+You can also specify the TLS version and the ciphers that you want to use:
 
 ```conf
 [web]
     tls version = 1.3
     tls ciphers = TLS_AES_256_GCM_SHA384:TLS_CHACHA20_POLY1305_SHA256:TLS_AES_128_GCM_SHA256
 ```
 
-case you do not specify these options, Netdata will use the highest options available on your system.
+If you do not specify these options, Netdata will use the highest options available on your system.
 
-An important note about the `tls version`, Netdata accepts all the TLS version as arguments (`1`, `1.1`, `1.2` and `1.3`), but
- only the last option is considered safe.
+While Netdata accepts all the TLS version as arguments (`1`, `1.1`, `1.2` and `1.3`), we recommend you use `1.3` for the most secure encryption.
```

Comment:
Two things WRT this that may need to be considered in the code:

* We should probably accept 1.0 and alias it to 1 for the protocol version, as that's what most other software does.
* It would be a very good idea to support a list of protocol versions instead of enforcing a specific one. Only allowing a specific protocol is potentially problematic from the perspective of client-side support (that is, it may not be possible to support all the clients a user wants to support if they are only able to select a single protocol).

Reply:
For OpenSSL we have only SSLv3 and TLSx.y and considering that the option is named `tls version` I think this is not a problem, case OpenSSL uses another protocol, we will have to deal with this. 
About the alias, I think it is a good point, I will bring it in few minutes. Thank you!

Reply:
There are enough differences though between TLS 1.2 and 1.3 that clients that only support up to 1.2 can't connect to a server that only supports 1.3. That's kind of the reason that there's a version distinction there.

And, unfortunately, there are still a lot of systems out there with clients that only support 1.2, or even just 1.1, because anything newer required a new version of OpenSSL with some rather problematic ABI compatibility issues.

Reply:
I understand this, among the tests that I did, I forced the `slave` to use `1.2` and the server with `1.3` and as expected it could not work. 

I did not add any additional information for the issue about the relationship between tls versions, because openssl always try to get the highest value between client and server, on the other hand, I was forcing to use an unique version of the protocol. I will let this flexible between the minimum and maximum available for our users do not have problems.